### DISCUSSION TITLE
how do i disable the changes ui ?

### DISCUSSION THREAD

**Discussion by MosheAzraf** [gitkraken/vscode-gitlens#2210] [Q&A] [answered]
hi,
i've created a react file, deleted all the unnecessary files that i dont need,
but gitlen keep showing me this king of "error" or annoying ui about the changes that i've made,
how do i completely disable that ?
![Screen Shot 2022-09-12 at 17 41 13](https://user-images.githubusercontent.com/39967669/189684189-b16b4681-e5b6-45e4-8587-32645f42536b.png)

**Comment by eamodio** [comment, marked as answer]
That feature is provided by VS Code itself, not GitLens. It is showing you the lines you've deleted from that file but haven't been staged or committed yet. You can control this via the `scm.diffDecorations` setting and other related settings. 

Hope that helps.

**Reply by MosheAzraf** [reply]
this seems to be working, thanks, i found scm.diffDecorations in the settings and disabled it
